R/page.R

```diff
@@ -96,13 +178,14 @@ validateCssPadding <- function(padding = NULL) {
 #'
 #' Create a dashboard layout with a full-bleed header (`title`) and [sidebar()].
 #'
-#' @inheritParams layout_sidebar
-#' @inheritParams page_fillable
 #' @param ... UI elements to display in the 'main' content area (i.e., next to
 #'   the `sidebar`). These arguments are passed to `layout_sidebar()`, which has
 #'   more details.
 #' @param title A string, number, or [htmltools::tag()] child to display as the
 #'   title (just above the `sidebar`).
+#' @inheritParams layout_sidebar
+#' @inheritParams page_fillable
+#' @inheritParams page_navbar
 #'
```

Comment:
Again, sorry to add more here, but now that I'm thinking about it, it might be nice to add a link to https://rstudio.github.io/bslib/articles/dashboards/index.html (both here and on `page_navbar()`'s page)?

Reply:
Great addition, thanks

Reply:
Done in d1b7743bff29aecbbbda823cc50bb083861cd0de